examples/components/utils/aggregatemodelweights/run.py

```diff
@@ -39,6 +40,18 @@ def get_arg_parser(parser=None):
     parser.add_argument(
         "--output", type=str, required=True, help="where to write the averaged model"
     )
+    parser.add_argument(
+        "--ancillary_files",
+        type=bool,
```

Comment:
I think you have to use distutils strtobool here.

Reply:
yep! Absolutely right